src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -592,11 +594,18 @@ public void selectDropdownByActualValue(WebElement element, String value) {
     }
 
     /**
-     * Returns the status message in the page. Returns "" if there is no
-     *         status message in the page.
+     * Returns a list of Texts of user status messages in the page.
+     * Returns a list containing "" if there is no status message in the page.
      */
-    public String getStatus() {
-        return statusMessage == null ? "" : statusMessage.getText();
+    public List<String> getTextsForAllUserStatusMessages() {
+        List<String> statusMessageTexts = new ArrayList<String>();
+        if (statusMessages.isEmpty()) {
+            statusMessageTexts.add("");
```

Comment:
???

Reply:
My reasoning was if there is no status message in the page, the method will contain a single string ""? (Now that I think of it, it is not a very good logic, a developer can just check if the array is empty or not) :sweat_smile: